parachain/pallets/erc20-app/.rustfmt.toml

```diff
@@ -0,0 +1 @@
+hard_tabs = true
```

Comment:
We don't currently use `rustfmt` as its incompatible with Substrate, so I'd rather we not check these in.

Reply:
Sure, I'll get rid of this. But I think it's compatible as long as the tabs are used, it just reformats code outside of macros.